### PR TITLE
Setup.py: Remove retro acronym

### LINKED ISSUE #178: Remove all instances of (code analysis application) from the repo

### PR DESCRIPTION
**Pull request by Divya063**
This removes all instances of (code analysis application) from the repo
Fixes https://github.com/coala/coala-quickstart/issues/178

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -17,8 +17,7 @@
 if __name__ == "__main__":
     setup(name='coala-quickstart',
           version='0.4.0',
-          description='A quickstart tool for for coala '
-                      '(Code Analysis Application)',
+          description='A quickstart tool for for coala ',
```
Comment (prnvdixit) [inline setup.py]: Can you also edit "tool ```for for``` coala" :smile: :+1:

Comment (Divya063) [reply setup.py]: oh sorry I didn't see that

**Comment by Divya063** [comment]
@prnvdixit squashed the commits and issue reference is working fine

**Comment by Divya063** [comment]
@prnvdixit Do you mean I should amend the commit?

**Comment by prnvdixit** [comment]
@Divya063 Yeah... that would be sufficient. Use ```git commit -a --amend``` for same. Also, refer to [Commit Guidelines](https://coala.io/commit) while doing same... :+1:

**Comment by Divya063** [comment]
@prnvdixit I guess this is fine now

**Comment by Makman2** [comment]
Please add inside the commit body `Closes <full-url-to-issue-fixed-by-your-commit>` ;)

File: setup.py [hunk 2]
```diff
@@ -17,8 +17,7 @@
 if __name__ == "__main__":
     setup(name='coala-quickstart',
           version='0.4.0',
-          description='A quickstart tool for for coala '
-                      '(Code Analysis Application)',
+          description='A quickstart tool for coala ',
```
Comment (Makman2) [inline setup.py]: redundant space after `coala` inside string

**Comment by Divya063** [comment]
@Makman2 Is this fine now?

**Comment by Divya063** [comment]
@prnvdixit Thanks for the help

**Comment by Divya063** [comment]
@prnvdixit I have never devoted so much time in just fixing the commit body. 😅

**Comment by prnvdixit** [comment]
ack 67d5809e09e55a6c51ae6e2e634a6f4ab75c6d5a

**Comment by Makman2** [comment]
@rultor merge

**Pull request merged**